configure.ac

```diff
@@ -1,6 +1,11 @@
     AC_INIT([suricata],[6.0.1-dev])
     m4_ifndef([AM_SILENT_RULES], [m4_define([AM_SILENT_RULES],[])])AM_SILENT_RULES([yes])
     AC_CONFIG_HEADERS([src/autoconf.h])
+
+    # FIXME: Undef HAVE_CONFIG_H since config.h is now not used
+    # Commit: 900f1522b444e8391250683d48855ceb3d23f225
+    AC_CONFIG_COMMANDS_PRE([test "X$DEFS" = X-DHAVE_CONFIG_H && DEFS=])
```

Comment:
Its related to this.  We still use HAVE_CONFIG_H, the file it loads is not autoconf.h instead of config.h though. Setting this appears to undef HAVE_CONFIG_H, sudo autoconf.h is not longer loaded, and its required.